CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# Changelog
+
+## 1.3.1 - 2024-07-03
+### Added
+- Added support for MailTrapClient ([20c88b7](https://github.com/audaciaconsulting/Audacia.Mail/pull/4/commits/20c88b7d4563a76b102e081fc988b18880419f94))
```

Comment:
This change was actually specifically support to pass in the `host` to the MailTrapClient, but comment implies that it added the MailTrapClient support itself (which was added several version prior).

Reply:
Thanks, I'll get that changed